src/test/java/dk/sdu/se_f22/sortingmodule/infrastructure/domain/SearchQueryTest.java

```diff
@@ -31,6 +31,20 @@ void clearRangeTest() {
 
     @Test
     void setPaginationTest() {
+        SearchQuery s = new SearchQuery();
+        int[] values = {1,2};
+        s.setPagination(values[0], values[1]);
+        assertArrayEquals(values, s.pagination);
+
+        values[0] = 654;
+        values[0] = 68469826;
+        s.setPagination(values[0], values[1]);
+        assertArrayEquals(values, s.pagination);
```

Comment:
Dine tests vil fejle.
Tænker du nok ville bruge [0] og [1] istedet for at assigne begge værdier til index [0].

Reply:
Funny enough - Så fejler min test ikke